### DISCUSSION TITLE
Display Remote Host Name Only

### DISCUSSION THREAD

**Discussion by risaw1981** [rustdesk/rustdesk#8756] [General]
Hi is there anyway to remove the Devices Windows Username from the names in list of devices. Example:

Administrator@File Server

I want to remove the "Administrator" as I only care about the name we have given it in Rustdesk so just display "File Server"

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/#hide-username-on-card

**Reply by risaw1981** [reply]
Great thanks
